locator/tablets.cc

```diff
@@ -103,9 +103,10 @@ tablet_migration_streaming_info get_migration_streaming_info(const locator::topo
             result.written_to = substract_sets(trinfo.next, tinfo.replicas);
             return result;
         case tablet_transition_kind::rebuild:
-            result.written_to.insert(trinfo.pending_replica);
+            assert(trinfo.pending_replica.has_value());
```

Comment:
This assert can kill majority of the cluster. Should be on_internal_error() or plain exception.